### ISSUE TITLE
[Discussion / Request] AutoHDR functionality support

### ISSUE THREAD

**Issue by JediRhymeTrix** [safijari/Reshadeck#11]
Hi,

Someone was able to get AutoHDR via Reshade working on the Deck OLED a few months back.

https://twitter.com/JavaidUsama/status/1763443358318428400

I was wondering if it was possible to implement this via this plugin so it's easier to enable it without having to leave game mode.

**Comment by JediRhymeTrix** [comment]
So, I found a way to make this work via the [Bash Shortcuts](https://github.com/Tormak9970/bash-shortcuts) plugin instead.

https://github.com/JediRhymeTrix/deck-toggle-autohdr

However, I'm keeping this issue open for discussion because I believe implementing this feature at the system level using this plugin (if possible) would offer more flexibility.

**Comment by JediRhymeTrix** [comment]
I had initially planned to build my own plugin for this, but then while looking up ways to detect the currently running game, I came across the Bash Shortcuts plugin which already does most of the work. So, I decided to build on top of that instead.

A standalone plugin would be the ideal solution, though. If nothing else, my solution could definitely be packaged a lot better than it currently is. Lol. I guess I could dive deeper into the Bash Shortcuts plugin and figure out a way to avoid redundancy as much as possible.

Anyways, thanks for your insight! I'm gonna go ahead and close this issue.
